writeit/settings.py

```diff
@@ -413,6 +413,10 @@
 # Logs every incoming email??
 INCOMING_EMAIL_LOGGING = 'None'
 
+# Should WriteInPublic handle bounces?
+FLAG_BOUNCED_CONTACTS = False
```

Comment:
How does settings.py work that this can be set in an environment? To me this code is always going to be `false`

Reply:
Yes - it's defaulting to False. This is to stop WriteInPublic from flagging bounces. If True, then WriteInPublic will work as normal.